scripts/cmake/vcpkg_configure_meson.cmake

```
@@ -33,67 +33,372 @@
 ##
 ## * [fribidi](https://github.com/Microsoft/vcpkg/blob/master/ports/fribidi/portfile.cmake)
 ## * [libepoxy](https://github.com/Microsoft/vcpkg/blob/master/ports/libepoxy/portfile.cmake)
-function(vcpkg_configure_meson)
-    # parse parameters such that semicolons in options arguments to COMMAND don't get erased
-    cmake_parse_arguments(PARSE_ARGV 0 _vcm "" "SOURCE_PATH" "OPTIONS;OPTIONS_DEBUG;OPTIONS_RELEASE")
+function(vcpkg_generate_native_file) #https://mesonbuild.com/Native-environments.html
```

```suggestion
function(vcpkg_internal_meson_generate_native_file) #https://mesonbuild.com/Native-environments.html
```

We want to note that these are _internal_, so you should be able to see if someone is using it (and could you make the same changes to all the other functions?)